App/authPopup.js

```diff
@@ -55,16 +65,6 @@ function signIn() {
         .then(handleResponse)
         .catch(error => {
             console.log(error);
-                
-            // Error handling
-            if (error.errorMessage) {
-                // Check for forgot password error
-                // Learn more about AAD error codes at https://docs.microsoft.com/en-us/azure/active-directory/develop/reference-aadsts-error-codes
-                if (error.errorMessage.indexOf("AADB2C90118") > -1) {
```

Comment:
Curious if all new sign-in/sign-up policies will have the built-in password reset by default? @nickgmicrosoft 

Would it make sense to continue demonstrating how to catch the error and call a dedicated password reset policy just for customers that are still using separate policies?

Reply:
I'll defer to you and @nickgmicrosoft here. Worst case I can keep one of the b2c samples the old way.